Gems/Terrain/Assets/Passes/TerrainPassRequest.azasset

```diff
@@ -0,0 +1,10 @@
+{
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "PassRequest",
+    "ClassData": {
+        "Name": "TerrainParentPass",
+        "TemplateName": "TerrainParentPassTemplate",
+        "Enabled": true
```

Comment:
Should be false I guess?
What if the parent is disabled but children passes are enabled?

Reply:
If the parent pass was disabled, the children passes' enable state won't matter. 

The default enable state of parent doesn't matter much here since even its enabled, the children are disabled. I can change to disable by default.